cmake/templates/BoostConfig.cmake.in

```diff
@@ -14,6 +14,24 @@ else()
   endfunction()
 endif()
 
+function(hunter_add_rt_library target_name)
+  include(CheckCXXSymbolExists)
+  check_cxx_symbol_exists("clock_gettime" "time.h" HUNTER_HAVE_CLOCK_GETTIME)
+  if(HUNTER_HAVE_CLOCK_GETTIME)
+      return()
+  endif()
+
+  include(CheckLibraryExists) # check_library_exists
+  check_library_exists(rt clock_gettime "" _boost_check_rt_library)
```

Comment:
The type and purpose of `_boost_check_rt_library` and `HUNTER_HAVE_CLOCK_GETTIME` are the same. Please unify the names so they will be consistent. Either `_boost_check_rt_library`/`_boost_have_clock_gettime` or `HUNTER_HAVE_CLOCK_GETTIME`/`HUNTER_CHECK_RT_LIBRARY`. However I prefer `_hunter_boost_check_rt_library` and `_hunter_boost_have_clock_gettime`.